_rules/important-letter-spacing-wide-enough-fsy8c7.md

```diff
@@ -0,0 +1,301 @@
+---
+id: fsy8c7
```

Comment:
While this is a significant update, I don't think the difference is significant enough that we should create them as new rules. I think I'd rather see them as updates to the existing rules.